### PR TITLE
Created users table

### PR DESCRIPTION
**Pull request by katiekeel**
As discussed in standup, some columns may be incorrect or need to be modified; I am thinking of location in particular. However this will give us a baseline to start working on the rest of this functionality.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  enum role: ["default", "admin"]
+
+  # belongs_to :location
+
+  # validates_presence_of :auth_token_linkedin,
```
Comment (DesTodo) [inline app/models/user.rb]: @katiekeel @yoasyo25 I noticed we have these User attributes in the migration and schema, and I wasn't sure if there is a reason the validations should be commented out. Is it okay to merge comments into development as long as they don't get merged to master?

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
-  get 'welcome/index'
 
+  resources :sessions, only: [:new, :create, :destroy]
```
Comment (DesTodo) [inline config/routes.rb]: @katiekeel @yoasyo25 It appears the sessions routes are repeated. I think we should delete line 3.

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+  get '/login', to: "sessions#new"
+  post '/login', to: "sessions#create"
+  delete '/logout', to: "sessions#destroy"
+
+  resources :users, except: [:index]
+
+  get 'welcome/index'
```
Comment (DesTodo) [inline config/routes.rb]: @katiekeel @yoasyo25 I think line 10 was introduced as a bug; seems like root is already set. I think we should remove line 10.

File: app/models/user.rb (hunk 1 above)
Comment (DesTodo) [reply app/models/user.rb]: @katiekeel @yoasyo25 Did you leave them out to make the initial seed data easier to handle?

**Review by DesTodo — COMMENTED** [review]
Overall looks great. Please review comments and make changes if necessary.

File: config/routes.rb (hunk 3 above)
Comment (katiekeel) [reply config/routes.rb]: I agree.

File: app/models/user.rb (hunk 1 above)
Comment (katiekeel) [reply app/models/user.rb]: Originally, we thought we would be able to pull user data from the LinkedIn API without a user logging in to LinkedIn, so we left those attributes on the user with default nil values. The validations being present still prevented us from getting the bcrypt solution up and running, so we commented them out for the time being. We can delete those validations and validations tests for now and re-add them later if we add those attributes back onto the user. I think we'll be able to do this if we implement some kind of button on the user profile page that allows a user to import their LinkedIn data.

File: config/routes.rb (hunk 2 above)
Comment (katiekeel) [reply config/routes.rb]: Yep 👍

**Comment by katiekeel** [comment]
I had to add some extra config for the Travis to work, but it passes now with our current test suite (aka one file with 3 model validations 😄)

**Comment by yoasyo25** [comment]
Thanks for updating the PR. This looks good.

**Pull request merged**